Examples/Elastix/par0001.txt

```diff
@@ -0,0 +1,72 @@
+// Description: elastixpaper, affine
```

Comment:
This file will need to be moved into Testing/Data/Baseline, hashed, uploaded to https://github.com/SimpleITK/SimpleITKData. It can then be referenced by  DATA{${SimpleITK_DATA_ROOT}/Baseline/ElastixTest_par0001.txt}.

Please let me know when it's ready and I can upload the data, and create the hash file in the PR.